### ISSUE TITLE
[Suggestion]Crystal mode for long jump

### ISSUE THREAD

**Issue by SStatikk** [DarkiBoi/konas-issues#94]
Add a crystal mode for long jump so its more bypassable, basically placing a crystal behind your back blowing it up and perfoming the jump

**Comment by Tux8068** [comment]
i don't think that would be good for some servers, because if you're longjumping away cus ur low health you will pop yourself
